packages/components/src/components/hds/advanced-table/th-context-menu.hbs

```diff
@@ -0,0 +1,17 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Dropdown class="hds-advanced-table__th-context-menu" ...attributes as |D|>
```

Comment:
[question] Should `@enableCollisionDetection` be set to true by default on this? Would that work properly within the scrollable content of the table?